lib/numbers_and_words/strategies/figures_converter/languages/cs.rb

```diff
@@ -0,0 +1,66 @@
+module NumbersAndWords
+  module Strategies
+    module FiguresConverter
+      module Languages
+        class Cs < Base
+          include Families::Latin
+
+          # methods in translations/cs.rb which can work with :gender or :ordinal
+          %i[ones teens tens tens_with_ones hundreds].each do |method_name|
+            define_method(method_name) do
+              super(internal_options)
+            end
+          end
+
+          %i[megs thousands].each do |method_name|
+            define_method(method_name) do
+              super(internal_options)
+            end
+          end
+
+          def zero
+            super(internal_options) unless maybe_remove_zero
+          end
+
+          def megs(*args)
+            super({ number: @figures.number_in_capacity(@current_capacity) }.merge(args.first || {}))
+          end
+
+          def gender
+            @current_capacity ||= 0
+            # @current_capacity = order of block of 3 digits, backwards
+
+            case @current_capacity
```

Comment:
I guess this `case` could be replaced with a multiple `if` statement, for instance:

```ruby
if @current_capacity.zero?
  options.gender.result
elsif @current_capacity > 2 && @current_capacity.odd?
  :female
else
  :male
end
```